python/cudf/cudf/core/column/struct.py

```diff
@@ -57,7 +58,9 @@ def to_arrow(self):
             pa_type, len(self), buffers, children=children
         )
 
-    def to_pandas(self, index: pd.Index = None, **kwargs) -> "pd.Series":
+    def to_pandas(
+        self, index: Optional[pd.Index] = None, **kwargs
+    ) -> "pd.Series":
```

Comment:
I think we can remove the quotes since we're importing pandas unconditionally here anyway.
```suggestion
    ) -> pd.Series:
```

Reply:
I guess I can, but maybe in a separate changeset?